samtranslator/model/api/api_generator.py

```diff
@@ -702,7 +864,9 @@ def to_cloudformation(
                 List[Resource],
                 Tuple[Resource],
                 List[LambdaPermission],
+                List[Resource],
```

Comment:
There's already another `List[Resource]`, you don't need this one again. (The `Union` here means that it could be of any of these types, so you don't need to repeat something that already exists)